docs/ai_actions/work_with_ai_actions.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Create new AI actions or modify existing ones to work faster and increase creativity.
+---
+
+# Work with AI actions
+
+AI actions define what results are available to editors in AI-enabled areas, such as, for example, the AI Assistant.
+If the AI Actions LTS update has been installed and configured in your application, and you have administrator permissions, you can reconfigure the existing AI actions, and create new ones.
```

Comment:
> and you have administrator permissions
Aren't there policies for granting access to managing AI actions ? I'd rather go for "required permissions" with a link to their documentation.